### PR TITLE
DTSAM-173: Refresh FTAs

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSAM-173

### Change description ###
Refresh FTAs

> NB: this PR must follow:
> * #1825

### Checklist
- [ ] Does this PR introduce a breaking change:  No

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/orgrolemapping/befta/OrgRoleMappingAmTestAutomationAdapter.java [hunk 1]
```diff
@@ -66,8 +68,18 @@ public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenario
...
+                return new TokenUtils().generateServiceToken(buildOrmSpecificConfig());
             default:
                 return super.calculateCustomValue(scenarioContext, key);
         }
     }
+
+    private UserTokenProviderConfig buildOrmSpecificConfig() {
+        UserTokenProviderConfig config = new UserTokenProviderConfig();
+        config.setMicroService("am_org_role_mapping_service");
+        config.setSecret(System.getenv("AM_ORG_S2S_SECRET"));
```
Comment (mattnayler) [inline src/functionalTest/java/uk/gov/hmcts/reform/orgrolemapping/befta/OrgRoleMappingAmTestAutomationAdapter.java:81]: I think this needs to be `AM_ORG_ROLE_MAPPING_SERVICE_SECRET`.

Comment (SivaKanukolluQA) [reply src/functionalTest/java/uk/gov/hmcts/reform/orgrolemapping/befta/OrgRoleMappingAmTestAutomationAdapter.java]: Updated the review comment.

**Review by mikebrownccd — DISMISSED** [review]
LGTM

**Review by abul-syed — DISMISSED** [review]
lgtm

**Review by mattnayler — DISMISSED** [review]
LGTM

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by mattnayler — APPROVED** [review]
re-approved after 36f1738bebe75d836986b3ca941cb65da1f42ed8

**Pull request merged**